theme/static/style/style.css

```diff
@@ -558,7 +558,6 @@ pre code {
         clear: both;
         font-size: 1.5rem;
         padding-left: 1.8rem;
-        padding-right: 1.8rem;
```

Comment:
This padding is actually needed because text on the right is very close to the edge of the screen. Drop this commit and rebase since I've implemented a different solution.

Reply:
Dropped and rebased, but the padding's still gone. Do I return it manually?

Reply:
Try fetching the latest source branch and rebase to it and this should be gone.